Vagrantfile

```diff
@@ -130,6 +130,14 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     testvm.vm.provision "python", type: "shell", inline: ubuntu_provision_python()
   end
 
+  config.vm.define "tester-amzn2" do |testvm|
```

Comment:
To be sort of consistent with the other Amazon image I suggest calling it tester-awslinux2.

Reply:
I renamed the box to awslinux2.